Userland/Games/Solitaire/main.cpp

```diff
@@ -40,9 +40,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     Config::pledge_domains({ "Games", "Solitaire" });
     Config::monitor_domain("Games");
 
-    TRY(Core::System::pledge("stdio recvfd sendfd rpath"));
+    TRY(Core::System::pledge("stdio recvfd sendfd rpath proc exec"));
 
     TRY(Core::System::unveil("/res", "r"));
+    TRY(Core::System::unveil("/bin/GamesSettings", "x"));
```

Comment:
Rather than unveiling and manually calling GameSettings, would it make sense to use LaunchServer, in the same was as for launching Help?

Reply:
I don't think that works for launching settings? That adds a URL handler to map `file:///path/to/Solitaire.md` to open with `Help`, whereas we don't have a way to open settings by URL

Reply:
Ah, yup. I shouldn't try and phone-review things when I just woke up. :sweat_smile: